source/StdAfx.h

```diff
@@ -78,11 +82,11 @@ typedef UINT64 uint64_t;
 #include <vector>
 #include <cassert>
 
-#include "windows.h"
+#include "linux/libwindows/windows.h"
```

Comment:
don't touch this. my `windows.h` is supposed to be a drop in replacement of the real one. if there is an ambiguity, it means there is a problem in the makefile.

Reply:
The ambiguity is that I didn't know it wasn't Microsoft's `windows.h` until I had spent several more hours trying to detangle defines.